Snabble/UI/Payment/QRCheckoutViewController.swift

```diff
@@ -86,8 +165,8 @@ final class QRCheckoutViewController: UIViewController {
 
     private func setupQrCode() {
         let qrCodeContent = self.process.paymentInformation?.qrCodeContent ?? "n/a"
-        self.qrCodeView.image = QRCode.generate(for: qrCodeContent, scale: 5)
-        self.qrCodeWidth.constant = self.qrCodeView.image?.size.width ?? 0
+        self.qrCodeView?.image = QRCode.generate(for: qrCodeContent, scale: 5)
+        self.qrCodeView?.widthAnchor.constraint(equalToConstant: self.qrCodeView?.image?.size.width ?? 0).isActive = true
```

Comment:
You add this constraint multiple times if the method is called over and over again. It would be better to add this constraint in `loadView` and use `usingVariable(_:)` to add it to an instance variable. Just change the constant as before